### ISSUE TITLE
Type '"replace"' is not assignable to type 'CustomizeRule'.

### ISSUE THREAD

**Issue by henrypoon** [survivejs/webpack-merge#147]
**Issue**

TS would complain "string" is not assignable to type 'CustomizeRule' when I write webpack config file into TS file. Should we export 'CustomizeRule' to allow users to import this enum?

☑️  Using updated webpack-merge
☑️  Using NodeJS v12.16.3, ts-node v9.0.0
☑️  Using MacOS

webpack.config.ts
```
export default mergeWithCustomize({
    customizeObject: customizeObject({
        entry: 'replace',
        output: 'append',
    })
})(webpackConfig, themeConfig);
```
![image](https://user-images.githubusercontent.com/17983160/92566849-2fee2400-f2c0-11ea-9119-89ad9b2b456e.png)

**Proposed**
Export 'CustomizeRule' and users could import it
e.g.
```
export default mergeWithCustomize({
    customizeObject: customizeObject({
        entry: CustomizeRule.Replace,
        output: CustomizeRule.Append,
    })
})(webpackConfig, themeConfig);
```

**Comment by bebraw** [comment]
@henrypoon Thanks for bringing this up! I'll expose `CustomizeRule`.

I guess another option would be to support simply strings instead of token.

**Comment by bebraw** [comment]
Solved in c4e293b7038 . I expose `CustomizeRule` now.
